test/api/searchResult.test.ts

```diff
@@ -0,0 +1,148 @@
+import { createMocks } from 'node-mocks-http';
+import { NextRequest, NextResponse } from 'next/server';
+
+import { GET } from '@/app/api/v1/searchResults/route';
+import { createManySearchResults } from '@/repositories/searchResult';
+import { createUser } from '@/repositories/user';
+
+import { user } from '../factories/user';
+import { searchResult } from '../factories/searchResult';
+
+describe('search result Get API:', () => {
```

Comment:
Should this be

```suggestion
describe('Search result get API:', () => {
```

As you are using [that format](https://github.com/nimblehq/ic-nextjs-prabesh-dahal/blob/3fdabf9f5438a15d6eb9e466d0f1fafde1f18e95/test/api/user.test.ts#L9) 🧓